maker/src/actor_system.rs

```diff
@@ -272,8 +268,8 @@ where
                 pong_address.clone(),
                 identify_listener_actor,
                 order,
-                rollover_v_1_0_0_addr.clone(),
-                rollover_v_2_0_0_addr.clone(),
+                rollover_deprecated_addr.clone(),
```

Comment:
🙃 maybe better to flip these so we always have `rollover_addr` first and the deprecated on second.